modules/ROOT/pages/audit-logging.adoc

```diff
@@ -29,7 +31,18 @@ Each log entry has a set of properties that provides information about the activ
 * *Type*: The type of the object on which the action is performed, for example, `Organization`
 * *Action*: The action associated with the object, for example, `Create`
 * *Object*: The name of the Object, for example, `foo`
-* *User*: The User who performed the action, for example, `johndoe`
+* *User*: The User who performed the action, for example, `johndoe` 
++
+[NOTE]
+--
+You may occasionally see an *Anypoint Staff* log entry. This log entry is made when the audit action was performed by a user who does not belong to your master organization. +
+These actions include when: 
```

Comment:
I don't think you should separate the object of the preposition (the stuff after "when") across a list & intro. How about `Anypoint Staff` entries are created for when these events occur:
* An internal or system process performs a task
* A MuleSoft Sales staff member changes entitlements for your organization
* A MuleSoft employee takes an action using a bearer token...